data/json/snippets/nightmares.json

```diff
@@ -8,6 +8,18 @@
       "You awaken from a nightmare about an unseen predator slowly killing you.",
       "You awaken from a nightmare about killing those you love with your bare hands.",
       "You awaken from a nightmare about being finally judged for your ill deeds.",
+      "You awaken from a nightmare,  recent events had not come to pass, and you were stuck in an office job.",
+      "You awaken from a nightmare,  this wasn't the end of the world… all those innocents you've murdered.",
+      "You awaken in a cold sweat,  relieved that you weren't being eaten alive by evil twinkies.",
+      "You awaken feeling hollow;  you dreamt of your friends and family,  only to realize your new reality is the nightmare.",
+      "You abruptly awake,  relief washes over you as you were no longer naked in the street,  and the zombies weren't laughing at you.",
+      "You feel something gently embrace you as you sleep.  You open your eyes,  but it's too dark to see,  you struggle to move,  but it has your arms and legs in a cocoon of darkness,  it has stolen your mouth,  and you cannot scream.  It starts to devour you whole,  starting with your feet,  unable to escape,  and unable to scream.  You take your last breaths.  Then you wake up,  it was just a dream.",
+      "Your in a meadow,  unsure of how you got here.  The sun is shining,  the birds are singing,  and children are playing;  all is peaceful.  So why do the hairs on your neck stand at attention?  Something is not quite right about this peace.  You turn around,  and the children have you surrounded.  The faces of the children have vanished,  replaced by a shark-like grin and beady black eyes.  They reach out with their pale,  claw-like hands.  However,  you cannot move,  you cannot escape…  you awake.",
```

Comment:
```suggestion
      "You are in a meadow,  unsure of how you got here.  The sun is shining,  the birds are singing,  and children are playing;  all is peaceful.  So why do the hairs on your neck stand at attention?  Something is not quite right about this peace.  You turn around,  and the children have you surrounded.  The faces of the children have vanished,  replaced by a shark-like grin and beady black eyes.  They reach out with their pale,  claw-like hands.  However,  you cannot move,  you cannot escape…  you awake.",
```

Reply:
I didn't think about the possible double messaging, I'll fix it after work.